snap_plugin/v1/plugin.py

```diff
@@ -203,6 +204,8 @@ class FlagType(Enum):
     value = 0
     toggle = 1
 
+class MissingRequiredArgument(Exception):
+   pass
```

Comment:
Please fix indentation and blank lines according to PEP 8 (indenting with 4 spaces, 2 blank lines between classes)